src/app/services/feelog/feelog.service.ts

```diff
@@ -18,7 +18,7 @@ export class FeelogService {
   }
 
   getFeeCodesAndDescriptions(code: string) {
-    let url = `${environment.apiUrl}/fee-codes`;
+    let url = `${environment.apiUrl}/fees`;
```

Comment:
it should say something like fees_api_url or fees.api.url rather than just apiUrl